docs/Using-Fleet/MDM-setup.md

```diff
@@ -7,41 +7,89 @@ In Fleet, MDM features are supported for Macs running macOS 12 (Monterey) and hi
 
 # Overview
 
-To use Fleet's MDM features you first first have to [deploy Fleet](../Deploying/Introduction.md) and [add your hosts to Fleet](./Adding-hosts.md).
-
 MDM features require Apple's Push Notification service (APNs) to control and secure Apple devices. This guide will walk you through how to generate and upload a valid APNs certificate to Fleet in order to use Fleet's MDM features.
 
 [Automated Device Enrollment](https://support.apple.com/en-us/HT204142) allows Macs to automatically enroll to Fleet when they are first set up. This guide will walk you through how to connect Apple Business Manager (ABM) to Fleet. Note that this is only required if you are using Automated Device Enrollment AKA Device Enrollment Program (DEP) AKA "Zero-touch."
 
-> Only users with the admin role in Fleet can complete these setups.
+# Requirements
+To use Fleet's MDM features you must have:
+1. A [deployed Fleet instance](../Deploying/Introduction.md)
+2. A Fleet user with the admin role
 
 ## Apple Push Notification service (APNs)
+Apple uses APNs to authenticate and manage interactions between Fleet and the host. 
+
+To connect Fleet to APNs, we will do the following steps:
+1. Generate four required files
+2. Generate an APNs certificate from APNs
```

Comment:
```suggestion
2. Generate an APNs certificate from Apple Push Certificates Portal
```
Minor nit: User goes to Apple Push Certificates Portal. Below screenshot is from here: https://support.apple.com/en-us/HT208643
![Screenshot 2023-04-03 at 10 08 14 AM](https://user-images.githubusercontent.com/47070608/229534397-4395f5cb-a0fd-45ad-bbd9-51eaaa458688.png)